data/campaigns/Eastern_Invasion/scenarios/13_Spoils_of_War.cfg

```diff
@@ -584,6 +588,8 @@
         {MOVE_UNIT side=1 42 4}
         [message]
             id=Owaec,Terraent
+            #po: TODO for 1.19? There's a cut-scene running up to this, but it's unclear what's happened until I read the WML and found that player's full starting roster is exactly one *horseman* for this scenario.
```

Comment:
I'm not sure if I like the idea of putting TODO notes in the translator's notes… this isn't the only one in here, either.

Reply:
> I'm not sure if I like the idea of putting TODO notes in the translator's notes… this isn't the only one in here, either.

I think it could be useful for warning translators that "hey this string might change; don't get too attached to your translation for it, and be aware that it might become fuzzy on you"

Reply:
Maybe that makes sense for example in that place where the wrong pronoun was used, but how does this TODO note indicate any upcoming change to the string?

Also, I think TODO notes just look bad in a place like this. Something could still be mentioned even without that blaring TODO sign.